plugins/casechange.md

```diff
@@ -46,12 +46,10 @@ tinymce.init({
 ```
 
 
-### Types of Formats
+## Types of Formats
 
 Currently, there are three types of formats available:
 
-![**Case Change**]({{site.baseurl}}/images/casechange.png)
```

Comment:
This image was deleted back in 63df78e0f87f798adbb988b43bb076eeba4a2cd5 and the reference in the enterprise page was removed, but we missed this one.

Reply:
actually, this one is kind-of funny to see rendered